compiler/pash_compilation_server.py

```diff
@@ -414,6 +429,16 @@ def run(self):
             self.parse_and_run_cmd(input_cmd)
         
         self.connection_manager.close()
+        if not self.parallelized_flag:
+            log("No parts of the input script were parallelized. Ensure commands are annotated for parallelization.")
```

Comment:
These messages need to be logged no matter the debug level, so they should be given a different level (the way it is now they will only be printed if we have -d 1. Also it would be good to prefix them with [PaSh Warning]. Here is some wordsmithing to make them a bit clearer too:
- `[PaSh Warning] No region of the script was parallelized. Maybe you are missing relevant annotations? Use `-d 1` for more info`.
- `[PaSh Warning] Some script regions were parallelized but their execution times were negligible (<1s). If your script takes a long time maybe annotations are missing from relevant regions. Use `-d 1` for more info.`